test/test_model.py

```diff
@@ -2423,14 +2589,11 @@ def test_unresolved_ingress_addresses(self):
 ]
 
 
-class TestModelBackend(unittest.TestCase):
-
-    def setUp(self):
-        self._backend = None
-
+class TestModelBackend:
     @property
     def backend(self):
-        if self._backend is None:
+        backend_instance = getattr(self, '_backend', None)
+        if backend_instance is None:
```

Comment:
Should we just instantiate `backend = _ModelBackend('myapp/0')` explicitly in the tests that need it, rather than using this "smart caching property"?

Reply:
The original test class has this mechanism; and I tested it without, it took 3 more seconds.